### ISSUE TITLE
Fix the link to PuDB in the TDD with Twisted doc

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7865]
|<img alt="mineo's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| mineo reported|
|-|-|
|Trac ID|trac#7865|
|Type|enhancement|
|Created|2015-04-23 13:08:55Z|
|Branch|https://github.com/twisted/twisted/tree/trial-tdd-link-7865|

The link is currently not treated as a link by RST (https://twistedmatrix.com/documents/current/core/howto/trial.html#dropping-into-a-debugger) because it's not preceded by a space. The attached patch adds that space.

Attachments:

* [trial-doc-pudb.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/a86/a8638133c52553d16188d9557995d9d6f1831829/8d96dd3f1e51338dcfd7ea41bc36076aa5630c51.patch) (850 bytes) - added by mineo on 2015-04-23 13:09:11Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__7865 7865
type__enhancement enhancement
reporter__mineo mineo
priority__normal normal
milestone__ 
branch__branches_trial_tdd_link_7865 branches/trial-tdd-link-7865
branch_author__adiroiban adiroiban
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1429794535232886 1429794535232886
changetime__1429796671618572 1429796671618572
version__None None
owner__mineo mineo

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_1"></a>|@adiroiban commented|
|-|-|

(In [44533]) Branching to trial-tdd-link-7865.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban set owner to mineo|
|-|-|

Thanks. Changes looks good.

Waiting for tests to pass and will merge.

Thanks again!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set status to `closed`|
|-|-|

(In [44535]) Merge trial-tdd-link-7865: Fix the link to PuDB in trial narrative documentation.

Author: mineo
Reviewers: adiroiban
Fixes: [#7865](https://github.com/twisted/twisted/issues/7865)
